packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -787,7 +805,45 @@ export async function startBrowser(
     ...webdriverOptions,
     ...wdioOptions,
   })) as CompassBrowser;
-  await browser.navigateTo('http://localhost:7777/');
+
+  if (TEST_ATLAS_CLOUD_EXTERNAL_URL) {
+    // Navigate to a 404 page to set cookies
+    await browser.navigateTo(`https://${TEST_ATLAS_CLOUD_EXTERNAL_URL}/404`);
```

Comment:
Why are wqe doing this if we're setting cookies from a file below?

Reply:
Ah, okay, I thought we're going on this page so that the set-cookie header from the backend can set some cookies, not so that the `browser.setCookies` call can work, got it!